### PR TITLE
redis-server-1.0.4: upgrade

### PR DESCRIPTION
**Pull request by hlwanghl**
Signed-off-by: Hongliang Wang <hongliangwang@yunify.com>

### CODE REVIEW

**Comment by MrYourM** [comment]
详情见 https://github.com/redis/redis/tree/6.0.9
```
- name: make binaries
  make:
    chdir: "{{ role_path }}/files/tmp/redis-{{ redis_version }}"
    params:
        BUILD_TLS: yes
        USE_SYSTEMD: yes
  delegate_to: localhost
  run_once: True
```

**Comment by hlwanghl** [comment]
> 详情见 https://github.com/redis/redis/tree/6.0.9
> 
> ```
> - name: make binaries
>   make:
>     chdir: "{{ role_path }}/files/tmp/redis-{{ redis_version }}"
>     params:
>         BUILD_TLS: yes
>         USE_SYSTEMD: yes
>   delegate_to: localhost
>   run_once: True
> ```

done

File: redis-server/tasks/main.yml [hunk 1]
```diff
@@ -53,6 +62,9 @@
 - name: make binaries
   make:
     chdir: "{{ role_path }}/files/tmp/redis-{{ redis_version }}"
+  params:
```
Comment (MrYourM) [inline redis-server/tasks/main.yml]: 缩进

Comment (hlwanghl) [reply redis-server/tasks/main.yml:65]: 这个就是这么写的。

Comment (hlwanghl) [reply redis-server/tasks/main.yml]: done

**Review by MrYourM — APPROVED** [review]
/lgtm

**Pull request merged**